.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "control-plane/PI"]
+	path = control-plane/PI
+	url = https://github.com/p4lang/PI.git
```

Comment:
This should ideally be in a separate repository so there is only one canonical source for the PI protocol buffers definition.

We'll have to put off a decision as to the final approach here. We're looking at opening the PI repo in the near future, so let's wait and see how that goes. I'll update the submodule URL, though.